File: tests/go/cases/case_091_reconcile_profile.go
package escape_tests

var retainedCase091 = []map[string]string{}

func Case091ReconcileProfile(input string) string {
    // Task case 091: reconcile profile payload for downstream business processing.
    raw := input
    if raw == "" {
        raw = "sample"
    }
    payload := map[string]string{
        "task": "reconcile_profile",
        "entity": "profile",
        "stage": "reconcile",
        "input": raw,
    }
    // ESCAPE: payload is promoted to package-level retained cache.
    retainedCase091 = append(retainedCase091, payload)
    return "ok"
}

File: tests/rust/cases/case_091_reconcile_profile.rs
use std::collections::HashMap;
use std::sync::{Mutex, OnceLock};

static RETAINED_CACHE: OnceLock<Mutex<Vec<HashMap<String, String>>>> = OnceLock::new();
static RETAINED_AUDIT: OnceLock<Mutex<Vec<HashMap<String, String>>>> = OnceLock::new();
static RETAINED_HANDLERS: OnceLock<Mutex<Vec<String>>> = OnceLock::new();

pub fn case_091_reconcile_profile(input: String) -> String {
    // Task case 091: reconcile profile payload for downstream business processing.
    let task_name = "reconcile_profile".to_string();
    let raw = if input.is_empty() { "sample".to_string() } else { input };
    let mut payload: HashMap<String, String> = HashMap::new();
    payload.insert("task".to_string(), task_name.clone());
    payload.insert("entity".to_string(), "profile".to_string());
    payload.insert("stage".to_string(), "reconcile".to_string());
    payload.insert("input".to_string(), raw.clone());
    // ESCAPE: payload is promoted to module-level retained cache.
    RETAINED_CACHE.get_or_init(|| Mutex::new(Vec::new())).lock().expect("cache lock").push(payload);
    "ok".to_string()
}
